qiskit/qpy/binary_io/value.py

```diff
@@ -123,9 +227,9 @@ def _read_parameter_expression(file_obj):
     if _optional.HAS_SYMENGINE:
         import symengine
 
-        expr = symengine.sympify(parse_expr(file_obj.read(data.expr_size).decode(common.ENCODE)))
+        expr_ = symengine.sympify(parse_expr(file_obj.read(data.expr_size).decode(common.ENCODE)))
```

Comment:
Heh, I didn't expect to ever have a conflict with this string. I should have written out `symbolic_expression` here or something

Reply:
Eh, I probably should have thought that it's probably a relatively common abbreviation before squatting the `expr` name for the module I added.